code/modules/mob/living/carbon/human/blood.dm

```diff
@@ -88,26 +90,46 @@ var/const/BLOOD_VOLUME_SURVIVE = 122
 				if(!pale)
 					pale = 1
 					update_body()
-					var/word = pick("dizzy","woozy","faint")
-					src << "<span class='warning'>You feel [word].</span>"
-				adjustOxyLoss((BLOOD_VOLUME_NORMAL - blood_volume) / 100)
+				if(prob(15))
+					add_medical_effect(/datum/medical_effect/shock)
```

Comment:
Wait what?  If I lose ONE unit of blood, there's a 15% chance I go into shock every second?